src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        bulldozers.add(new Bulldozer()); // Додаємо новий Bulldozer до списку
```

Comment:
The `get` method currently adds only one `Bulldozer` to the list. If the task requires producing multiple bulldozers, consider adding more instances to the list.